pkg/network/tracer/connection/kprobe/tracer_test.go

```diff
@@ -305,23 +306,32 @@ func TestDefaultKprobeMaxActiveSet(t *testing.T) {
 	t.Run("CO-RE", func(t *testing.T) {
 		cfg := config.New()
 		cfg.EnableCORE = true
+		cfg.EnableRuntimeCompiler = false
 		cfg.AllowRuntimeCompiledFallback = false
+		cfg.AllowPrebuiltFallback = false
 		_, _, _, err := LoadTracer(cfg, manager.Options{DefaultKProbeMaxActive: 128}, nil, nil)
```

Comment:
I believe I changed the default maxActive to 512 a few months ago. do these tests work?

Reply:
They should; they just check if the option is passed in correctly to the tracer creator functions.